src/rpc/server/browserContextDispatcher.ts

```diff
@@ -18,17 +18,32 @@ import * as types from '../../types';
 import { BrowserContextBase, BrowserContext } from '../../browserContext';
 import { Events } from '../../events';
 import { Dispatcher, DispatcherScope, lookupNullableDispatcher, lookupDispatcher } from './dispatcher';
-import { PageDispatcher, BindingCallDispatcher } from './pageDispatcher';
-import { PageChannel, BrowserContextChannel, BrowserContextInitializer } from '../channels';
+import { PageDispatcher, BindingCallDispatcher, WorkerDispatcher } from './pageDispatcher';
+import { PageChannel, BrowserContextChannel, BrowserContextInitializer, CDPSessionChannel } from '../channels';
 import { RouteDispatcher, RequestDispatcher } from './networkDispatchers';
 import { Page } from '../../page';
+import { CRBrowserContext } from '../../chromium/crBrowser';
+import { CDPSessionDispatcher } from './cdpSessionDispatcher';
+import { Events as ChromiumEvents } from '../../chromium/events';
 
 export class BrowserContextDispatcher extends Dispatcher<BrowserContext, BrowserContextInitializer> implements BrowserContextChannel {
   private _context: BrowserContextBase;
 
   constructor(scope: DispatcherScope, context: BrowserContextBase) {
+    // Chromium-specific.
+    let backgroundPages: PageDispatcher[] = [];
+    if ((context as any).backgroundPages)
```

Comment:
contextbase has a browser base that has a browser type that has a name :)

Reply:
Plumbed.